packages/rest/src/rest.server.ts

```diff
@@ -561,6 +555,10 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._routerForStaticAssets.use(path, express.static(rootDir, options));
   }
 
+  get staticRouter() {
```

Comment:
Please don't, see my comment above.

Reply:
I agree. I'm playing with various options here to surface the problems.